iocore/net/P_UnixNet.h

```diff
@@ -243,32 +244,35 @@ class NetHandler : public Continuation, public EThread::LoopTailHandler
   uint32_t active_queue_size = 0;
 
   /// configuration settings for managing the active and keep-alive queues
-  struct Config {
-    uint32_t max_connections_in                 = 0;
-    uint32_t max_connections_active_in          = 0;
-    uint32_t inactive_threshold_in              = 0;
-    uint32_t transaction_no_activity_timeout_in = 0;
-    uint32_t keep_alive_no_activity_timeout_in  = 0;
-    uint32_t default_inactivity_timeout         = 0;
+  enum CONFIG {
+    max_connections_in,
+    max_connections_active_in,
+    inactive_threshold_in,
+    transaction_no_activity_timeout_in,
+    keep_alive_no_activity_timeout_in,
+    default_inactivity_timeout,
   };
+  static constexpr auto CONFIG_COUNT = static_cast<int>(CONFIG::default_inactivity_timeout) + 1;
```

Comment:
Fragile , some could add a new config value to the end and not notice this.

Reply:
I did that originally - I decided I liked have a non-valid enum less than having this dependency.